### ISSUE TITLE
[Feature Request] Push button should consider multiple selected branches

### ISSUE THREAD

**Issue by kapsiR** [ForkIssues/TrackerWin#2352]
## Description
Would be very helpful if the Push button considers multiple selected branches, like right-click and "Push X branches to 'origin'"

**Comment by DanPristupov** [comment]
You can select multiple branches on sidebar, right click and push them at once using context menu.

**Comment by kapsiR** [comment]
Yes I know that. It would be super cool to do this with just one click on the Push button 😊 (or even Quick Push)
